Ginger/Ginger/ALM/JIRA/TreeViewItems/JiraZephyrCycleTreeItem.cs

```diff
@@ -0,0 +1,80 @@
+#region License
+/*
+Copyright © 2014-2019 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using Ginger.SolutionWindows.TreeViewItems;
+using GingerWPF.UserControlsLib.UCTreeView;
+using GingerCore;
+using System;
+using System.Collections.Generic;
+using System.Windows.Controls;
+using Amdocs.Ginger.Common;
+using amdocs.ginger.GingerCoreNET;
+using GingerCore.ALM.Qtest;
+
+namespace Ginger.ALM.JIRA.TreeViewItems
+{
+    public class JiraZephyrCycleTreeItem : TreeViewItemBase, ITreeViewItem
+    {
+        public BusinessFlow QCExplorer { get; set; }
+        public List<ITreeViewItem> currentChildrens = null;
+
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public string VersionId { get; set; }
+
+        Object ITreeViewItem.NodeObject()
+        {
+            return null;
+        }
+
+        public JiraZephyrCycleTreeItem()
```

Comment:
Codacy found an issue: [Remove this redundant constructor.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)